packages/cosmic-swingset/src/launch-chain.js

```diff
@@ -175,21 +175,27 @@ export async function launch(
     schedulerBlockTimeHistogram.record((now - blockStart) / 1000);
   }
 
-  let bootstrapBlock;
+  async function bootstrapBlock(blockTime) {
+    controller.writeSlogObject({
+      type: 'cosmic-swingset-bootstrap-block-start',
+      blockTime,
+    });
+    // This is before the initial block, we need to finish processing the
+    // entire bootstrap before opening for business.
+    await crankScheduler(Infinity);
```

Comment:
@mhofman Be aware, here's another call site that wants a `runPolicy`, and this one wants a "give it as much time as it wants" option.

Reply:
Yup, makes it a lot more straightforward actually.